src/index.html

```diff
@@ -16,17 +16,30 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      action="try-los-angeles"
```

Comment:
Ensure that the `action` attribute value 'try-los-angeles' is a valid URL or path. If it's a placeholder, update it to a valid destination.